modules/netflow/README.md

```diff
@@ -0,0 +1,3 @@
+# Netflow Module README
+
+## Documentation here
```

Comment:
Can you add all the `var.netflow` overrides we expose in the README? will be easier to test